scripts/run-ah-linter.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -e
+
+mkdir -p ah
+AH_VERSION=$(curl -s https://api.github.com/repos/artifacthub/hub/releases | \
+            jq '.[] | select(.prerelease==false) | .tag_name' | sort -V | tail -n1 | tr -d '"')
```

Comment:
Use `jq` solution without piping into sort, tail and tr?

Reply:
fixed